subscriptions/models.py

```diff
@@ -58,7 +60,7 @@ def get_expected_next_sequence_number(self, end_date=None):
             end_date = now()
         set_max = self.messageset.get_messageset_max(self.lang)
         runs = self.schedule.get_run_times_between(self.created_at, end_date)
-        count = len(runs)
+        count = len(runs) + (self.initial_sequence_number - 1)
```

Comment:
why `-1`?
If a subscription should have run 5 times and it started with `initial_sequence_number=5` then we'd expect `next-sequence_number` it to be at 10 right? (the 10th message should be run next) but this will put it at 9
And, if the same subscription started with `initial_sequence_number=1` then we'd expect it to be at 6 but this would put it at 5